Makefile.am

```diff
@@ -14,3 +15,18 @@ coverage: check
 	@echo -e "libica coverage";
 	@echo -e "---------------\n";
 	cd ${top_builddir}/src && gcov .libs/*.gcda
+
+if ICA_FIPS
+check: all
+	@echo make all and all-local before make check because of .hmac file
+
+all-local:
+	openssl dgst -sha256 -mac hmac -macopt hexkey:00000000 src/.libs/libica.so.$(VERSION) | sed -e 's/^.* //' > src/.libs/.libica.so.$(MAJOR).hmac
+
+install-data-hook:
+	cp src/.libs/.libica.so.$(MAJOR).hmac /usr/local/lib/
```

Comment:
Shouldn't we use libdir instead of hard-coding the install path?

Reply:
hmm yes, I agree, but $(LIBDIR) is not set here .... maybe we can pass it via libica.spec ...

Reply:
ok, I found out that the variable is $(libdir) and not $(LIBDIR)! So I will change the hard coded /usr/local/lib to $(libdir).